skipper-mac.yaml

```diff
@@ -0,0 +1,35 @@
+registry: quay.io
+build-container-image: assisted-service-build
+
+containers:
+    assisted-service-build: Dockerfile.assisted-service-build
+volumes:
+    - $HOME/.cache/go-build:/go/pkg/mod
+    - $HOME/.cache/golangci-lint:$HOME/.cache/golangci-lint
+    - $HOME/.minikube:$HOME/.minikube
+    - $HOME/.docker/:$HOME/.docker
+env:
```

Comment:
I'm guessing this is a duplicate of the skipper.yaml
Consider extracting it to a separate file and use the env-file option
https://github.com/Stratoscale/skipper#environment-variables